src/main/webapp/src/components/CourseDetails.vue

```diff
@@ -206,20 +253,117 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.label {
-    width: 7rem;
-    font-weight: 700 !important;
-    float: left !important;
+// page container
+.course-details-container {
+    margin: auto;
+    max-width: 1200px;
+}
+.course-details-main-container {
+    display: flex;
+    justify-content: space-between;
+}
+// buttons container
+.course-details-nav {
+    margin-bottom: $xxl-space;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.nav-item > button:first-child {
+    margin-right: $s-space;
+}
+.button-icon {
+    padding-right: $xs-space;
+}
+// course content
+.course-details-content-container {
+    max-width: 640px;
+}
+.course-details-image-container {
+    margin-bottom: $l-space;
+    width: 640px;
+    min-height: 120px;
+    max-height: auto;
+    padding: 3px;
+}
+.course-img {
+    max-width: 100%;
+    height: auto;
+}
+.course-details-content-date {
+    margin-bottom: $xs-space;
+}
+.course-details-content-title {
+    font-size: $xl-font;
+    font-weight: $normal;
+    margin-bottom: $l-space;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 5;
+    -webkit-box-orient: vertical;
```

Comment:
Warum hier nur die webkit Werte?

Noch besser keine browser spezifischen Werte in eine scss Funktion auslagern und hier nur den zielstandard aufschreiben.

Reply:
unsere app Anforderung ist chrome und firefox zu unterstützen. -webkit funktioniert in beide :)